ci/do_ci.sh

```diff
@@ -252,6 +261,12 @@ elif [[ "$CI_TARGET" == "bazel.coverity" ]]; then
      "${ENVOY_BUILD_DIR}"/envoy-coverity-output.tgz \
      "${ENVOY_DELIVERY_DIR}"/envoy-coverity-output.tgz
   exit 0
+elif [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
```

Comment:
@lizan do we want this in CircleCI or https://github.com/envoyproxy/envoy/blob/master/.azure-pipelines/linux.yml#L4? This test will take at least as long as anything that builds `//source/server/...`, since we do things like validate a server on a bootstrap, but there are relatively few (~dozen) actual tests.

Reply:
Let's do it in Azure